guide/layouts/default.haml

```diff
@@ -0,0 +1,12 @@
+!!! 5
+%html{lang: "fr", "data-fr-scheme" => "system"}
```

Comment:
nitpick: I tend to use symbols even for strings that contain dashes like `"data-fr-scheme": "system"` but I guess if the linter is happy with this it's completely fine

Reply:
forgot it works this way too, will amend and merge